sql/log_event.cc

```diff
@@ -2409,12 +2409,11 @@ void Log_event::print_base64(IO_CACHE *file, PRINT_EVENT_INFO *print_event_info,
     }
 
     if (print_event_info->inside_binlog) {
-      if (ev)
-        print_event_info->verbose_events.push_back(ev);
+      if (ev) print_event_info->verbose_events.push_back(ev);
```

Comment:
Do we really need code formatting changes in this commit?

Reply:
May be this file is touched for some other reason. I will try to extend clang-format commit. That commit didn't cover all files.